dissect/target/filesystems/tar.py

```diff
@@ -69,9 +69,9 @@ def detect(fh) -> bool:
         finally:
             fh.seek(offset)
 
-    def get(self, path) -> TarFilesystemEntry:
+    def get(self, path: str, *args, **kwargs) -> TarFilesystemEntry:
```

Comment:
I think it is nicer to pass the variables explicitly like:
```suggestion
    def get(self, path: str, relentry: FilesystemEntry = None) -> TarFilesystemEntry:
```